test/lint/lint-filenames.sh

```diff
@@ -10,7 +10,11 @@
 export LC_ALL=C
 
 EXIT_CODE=0
-OUTPUT=$(git ls-files -- "*.cpp" "*.h" "*.py" "*.sh" | grep -vE '^[a-z0-9_./-]+$' | grep -vE 'src/(secp256k1|univalue)/')
+OUTPUT=$(git ls-files --full-name -- "*.[cC][pP][pP]" "*.[hH]" "*.[pP][yY]" "*.[sS][hH]" \
+| grep -vE '^[a-z0-9_./-]+$' \
+| grep -vE '^src/(secp256k1|univalue)/' \
+)
```

Comment:
Nit: Please format it like this to make it consistent with the rest of the lint shell scripts:

```
OUTPUT=$(git ls-files --full-name -- "*.[cC][pP][pP]" "*.[hH]" "*.[pP][yY]" "*.[sS][hH]" | \
    grep -vE '^[a-z0-9_./-]+$' | \
    grep -vE '^src/(secp256k1|univalue)/')
```